tests/test_utils.py

```diff
@@ -107,15 +137,15 @@ def test_ssl_context(self, m_ssl, m_os):
     async def test_request_token(self, m):
         """Test token request."""
         # Test token received
-        m.post('https://aai.org/token', status=200, payload={'access_token': 'secret'})
+        m.post('https://login.elixir-czech.org/oidc/token', status=200, payload={'access_token': 'secret'})
```

Comment:
the url can be mocked via environment variable: `URL_TOKEN`
use https://docs.python.org/3.7/library/test.html#test.support.EnvironmentVarGuard.set in python to mock it

example here: https://github.com/CSCfi/beacon-python/blob/master/tests/test_app.py#L92-L102
Do not forget to unset it and this applies for all other similar URLs in tests, mock them, do not use real URLs